src/main/java/jep/MainInterpreter.java

```diff
@@ -258,7 +264,9 @@ public static void setInitParams(PyConfig config) throws IllegalStateException {
      *             if called after the Python interpreter is initialized
      * 
      * @since 3.7
+     * @deprecated Use {@link PyConfig#setArgv(String...)} instead.
```

Comment:
How confident are you that this new alternative will have the same behavior?

Reply:
For the primary goal of setting `sys.argv` for shared modules I am confident that the `TestSharedArgv` classes verifying the behavior is the same. I am not quite as confident that there are no other side effects. Both the old call to `PySys_SetArgvEx` and the new call to `PyConfig_SetBytesArgv` have documented optional side affects to set the path or parse the command line for options however in both cases we are setting it up to not do anything extra so as far as I can tell they are doing the same thing.